cmd/vic-machine/upgrade/upgrade.go

```diff
@@ -154,35 +154,36 @@ func (u *Upgrade) Run(clic *cli.Context) (err error) {
 	log.Infof("VCH ID: %s", vch.Reference().String())
 
 	if u.ResetInProgressFlag {
-		if err = vch.SetVCHUpdateStatus(ctx, "false"); err != nil {
-			log.Error("Failed to reset UpdateInprogress flag")
+		if err = vch.SetVCHUpgradeStatus(ctx, false); err != nil {
+			log.Error("Failed to reset UpgradeInprogress flag")
 			log.Error(err)
 			return errors.New("upgrade failed")
 		}
-		log.Infof("Reset UpdateInProgress flag successfully")
+		log.Infof("Reset UpgradeInProgress flag successfully")
 		return nil
 	}
 
-	upgrading, err := vch.GetVCHUpdateStatus(ctx)
+	upgrading, err := vch.VCHUpgradeStatus(ctx)
 	if err != nil {
-		log.Error("Unable to determine if upgrade/update is in progress")
+		log.Error("Unable to determine if upgrade/configure is in progress")
 		log.Error(err)
 		return errors.New("upgrade failed")
 	}
 	if upgrading {
-		log.Error("Upgrade failed: another upgrade/update operation is in progress")
+		log.Error("Upgrade failed: another upgrade/configure operation is in progress")
+		log.Error("Use --resetInProgressFlag to reset the VCH upgrade/configure status")
```

Comment:
still need to mention, if there is no other upgrade or configure process running, use ...